lib/AST/Decl.cpp

```diff
@@ -3083,6 +3084,14 @@ TypeRepr *ValueDecl::getOpaqueResultTypeRepr() const {
     returnRepr = SD->getElementTypeRepr();
   }
 
+  return returnRepr;
+}
+
+TypeRepr *ValueDecl::getOpaqueResultTypeRepr() const {
+  TypeRepr *returnRepr = nullptr;
+
+  returnRepr = this->getResultTypeRepr();
```

Comment:
Nit: `TypeRepr *returnRepr` doesn't need to be initialized with `nullptr` first anymore, you can just write
`auto *returnRepr = getResultTypeRepr();`